snuba/datasets/entities/transactions.py

```diff
@@ -111,12 +121,49 @@ def select_storage(
         return StorageAndMappers(storage, self.__mappers)
 
 
+class TransactionsV2QueryStorageSelector(QueryStorageSelector):
+    def __init__(self, mappers: TranslationMappers) -> None:
+        self.__transactions_table = get_writable_storage(StorageKey.TRANSACTIONS_V2)
+        # TODO: Register TRANSACTIONS_V2_RO and then remove comment
+        # self.__transactions_ro_table = get_storage(StorageKey.TRANSACTIONS_V2)
+        self.__mappers = mappers
+
+    def select_storage(
+        self, query: Query, request_settings: RequestSettings
+    ) -> StorageAndMappers:
+        # TODO: Register ERRORS_V2_RO and support the ro storage
```

Comment:
This is different for transactions, you do not need a selected a storage for transactions_v2 
just use `SingleStorageQueryPlanBuilder` at line 177. There is only one storage, no selection to be done.

Reply:
got it